src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck started work");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Truck stopped work");

```

Comment:
Similarly, the message 'Truck stopped work' should be more informative and reflect the specific action the Truck has ceased performing.